### ISSUE TITLE
Failed to create view for 'GraphView' 

### ISSUE THREAD

**Issue by rkelley4-chwy** [yWorks/yfiles-jupyter-graphs#64]
**Describe the bug**
When running the example code from `https://pypi.org/project/yfiles-jupyter-graphs/`, I'm getting the following error. 
```
Failed to create a view for 'GraphView' from module 'yfiles-jupyter-graphs' with model 'GraphModel' from module 'yfiles-jupyter-graphs'
```

**To Reproduce**
1. starting with python 3.11.9
2. create a virtualenv: 
```
python -m venv .venv
source .venv/bin/activate
pip install --upgrade pip
pip install jupyterlab
pip install yfiles-jupyter-graphs
```
3. start juypter-lab
4. run the following code:
```
from yfiles_jupyter_graphs import GraphWidget
GraphWidget()
```

8. See error:
```
[Open Browser Console for more detailed log - Double click to close this message]
Failed to create view for 'GraphView' from module 'yfiles-jupyter-graphs' with model 'GraphModel' from module 'yfiles-jupyter-graphs'
TypeError: Cannot read properties of undefined (reading '@jupyterlab/apputils-extension:themes')
    at http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:71511
    at a (http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:71535)
    at s (http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:71668)
    at t.updateThemeClass (http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:71911)
    at N. (http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:94146)
    at Generator.next ()
    at http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:90575
    at new Promise ()
    at i (http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:90320)
    at N.render (http://localhost:8889/lab/extensions/yfiles-jupyter-graphs/static/976.ba05290df1e07dfdfa16.js?v=ba05290df1e07dfdfa16:1:94042)
```

**Expected behavior**
I expected a blank `GraphWidget` to display. 

**Screenshots**
![image](https://github.com/user-attachments/assets/a1faf6ca-5148-4242-93c1-7d67905ac7c2)

**Desktop (please complete the following information):**
 - OS: MacOS Sonoma 14.7.2 (23H311) 
 - Browser: Chrome 131.0.6778.205 (Official Build) (arm64) and Firefox 133.0.3 (aarch64)

**Comment by fskpf** [comment]
Thanks for the report. I was able to reproduce it in a new environment.

It looks like the update from JupyterLab 4.2 to 4.3 broke it. We will release an update as soon as possible.

For now, it should work to install the latest JupyterLab 4.2 version, i.e.
```
pip install jupyterlab==4.2.6
```
Then run `jupyter lab` again and the error should not happen anymore. If in doubt, check the installed JupyterLab version with `pip show jupyterlab`.

**Comment by fskpf** [comment]
Fixed in [v1.10.1](https://pypi.org/project/yfiles-jupyter-graphs/1.10.1/).
